.circleci/config.yml

```diff
@@ -147,6 +147,7 @@ jobs:
             ulimit -u 20000
             ulimit -n 20000
             cargo +<< pipeline.parameters.nightly-toolchain >> -Zpackage-features test --all --verbose --release lifecycle -- --ignored --nocapture
+            cargo +<< pipeline.parameters.nightly-toolchain >> -Zpackage-features test --features single-threaded --release checkout_cores -- --test-threads=1
```

Comment:
this is missing the `--all` package specificer

Reply:
Correct, that's the purpose of this PR.  The all is performed, followed by the single checkout_cores test that is isolated.

Reply:
but it still needs to select a package?

Reply:
The CI log shows what I see locally, which is the single test running (after the all): https://app.circleci.com/pipelines/github/filecoin-project/rust-fil-proofs/2838/workflows/c44d3b1c-e69e-43cc-8c06-6c4e533ccf92/jobs/62773

Reply:
Ok, adding the package is more direct, I'll change that (just tried locally).